upi/aws/cloudformation/05_cluster_master_nodes.yaml

```diff
@@ -12,19 +12,6 @@ Parameters:
   RhcosAmi:
     Description: Current Red Hat Enterprise Linux CoreOS AMI to use for bootstrap.
     Type: AWS::EC2::Image::Id
-  AutoRegisterDNS:
-    Default: "yes"
-    AllowedValues:
-    - "yes"
-    - "no"
-    Description: Do you want to invoke DNS etcd registration, which requires Hosted Zone information?
-    Type: String
-  PrivateHostedZoneId:
-    Description: The Route53 private zone ID to register the etcd targets with, such as Z21IXYZABCZ2A4.
-    Type: String
-  PrivateHostedZoneName:
-    Description: The Route53 zone to register the targets with, such as cluster.example.com. Omit the trailing period.
-    Type: String
```

Comment:
We cannot remove these, as the CI jobs need to be able to run against multiple versions of OpenShift. Instead, we should be setting an unused default value and changing the description to indicate that they are no longer used.

See https://github.com/openshift/installer/commit/e11e2001030349f54315ba42cf969ac54aa9af9d as an example.

Reply:
Hi @staebler, sounds like a good plan, can you suggest the proper modification here? this patch is a bit old and I lost track of it, thanks.

Reply:
https://github.com/openshift/installer/pull/4795#discussion_r830301432